coalahtml/_coalahtml/app/controllers/files.js

```diff
@@ -33,7 +33,31 @@ angular.module('coalaHtmlApp')
     };
     $scope.fileName = $routeParams.fileName || $rootScope.ROOTHOME;
     parseCoalaProject($scope.fileName);
-    $scope.basename = function(path_name){
+
+    $scope.basename = function(path_name) {
+      path_name = path_name || $rootScope.ROOTHOME;
```

Comment:
`path_name` is `undefined` . It happens the first time page file tab is loaded. There is no fileName passed. For this case default is root.